traffic_portal/app/src/common/api/DeliveryServiceUriSigningKeysService.js

```diff
@@ -17,17 +17,31 @@
  * under the License.
  */
 
-var DeliveryServiceUriSigningKeysService = function(Restangular) {
+var DeliveryServiceUriSigningKeysService = function($http, ENV) {
 
 	this.getKeys = function(id) {
```

Comment:
it turns out "id" is really the "xmlId". want to rename this variable for clarity?